Revit_Core_Engine/Convert/FromRevit.cs

```diff
@@ -47,6 +47,13 @@ public static partial class Convert
         /****      Convert Revit elements to BHoM       ****/
         /***************************************************/
 
+        [Description("Converts a Revit ProjectInfo to a BHoM object based on its discipline.")]
```

Comment:
I would ctrl+h 'based on its discipline' to 'based on the requested engineering discipline'